<?xml version="1.0"?>
<!--

MIT License Agreement

Description of the jackal workspace for the multi-agent payload transport project.

\authors Swapnil Kalhapure <kalhapure.swapnil@gmail.com>

\maintainer Swapnil Kalhapure <kalhapure.swapnil@gmail.com>

-->
<launch>

    <rosparam param="jackal2/jackal_velocity_controller/left_wheel">[jackal2/front_left_wheel, jackal2/rear_left_wheel] </rosparam>
    <rosparam param="jackal2/jackal_velocity_controller/right_wheel">[jackal2/front_right_wheel, jackal2/rear_right_wheel]</rosparam>

    <!-- Spawn the robots. -->
    <include file="$(find multi_jackal_base)/launch/jackal_base.launch">
        <arg name="ns" value="jackal2" />
        <arg name="ons1" value="jackal0" />
        <arg name="ons2" value="jackal1" />
        <arg name="config" value="sheffield_robots" />
        <arg name="config_id" value="0" />
        <arg name="x" value="-0.55" />
        <arg name="y" value="-1.2" />
        <arg name="yaw" value="-1.57" />
        <arg name="use_global_tf" value="true" />
    </include> 
</launch>